pkg/api/handler/internal/TxFinder/tx_finder.go

```diff
@@ -48,6 +54,9 @@ func New(th metamorph.TransactionHandler, pc *config.PeerRPCConfig, w *woc_clien
 }
 
 func (f Finder) GetRawTxs(ctx context.Context, source validator.FindSourceFlag, ids []string) ([]validator.RawTx, error) {
```

Comment:
1. Shouldn't we include `TransactionHandler.GetTransactions()` in the trace?
2. I think adding multiple child spans with the same name won’t help us identify which part of the code is causing problems. But perhaps I’m missing the bigger picture.